pkg/metrics/object.go

```diff
@@ -254,3 +270,15 @@ func (m objectServiceMetrics) AddPutPayload(ln int) {
 func (m objectServiceMetrics) AddGetPayload(ln int) {
 	m.getPayload.Add(float64(ln))
 }
+
+func (m objectServiceMetrics) ChangeBy(shardID string, delta int) {
```

Comment:
`Change` what? Could you add `ObjectCounter` to the name?

Reply:
removed, only `AddToObjectCounter(shardID string, delta int)` and `SetObjectCounter(shardID string, v uint64)` are kept on the metrics side; wrappers still have `Inc*`, `Dec*` methods